src/Observable.ts

```diff
@@ -13,9 +13,13 @@ export interface Subscribable<T> {
   subscribe(observer: Observer<T>): Subscription;
 }
 
-export type SubscribableOrPromise<T> = Subscribable<T> | Promise<T>;
-export type ArrayOrIterator<T> = Iterator<T> | ArrayLike<T>;
-export type ObservableInput<T> = SubscribableOrPromise<T> | ArrayOrIterator<T>;
+export interface Observablesque<T> {
+  [Symbol.observable](): Subscribable<T>;
```

Comment:
You know what? Scratch that. Don't worry about it, for the purposes of the internals of our library where this is being used, we don't care.